packages/edit-post/src/components/sidebar/post-status/index.js

```diff
@@ -62,14 +62,14 @@ export default function PostStatus() {
 					<>
 						<PostVisibility />
 						<PostSchedulePanel />
-						<PostTemplate />
 						<PostURLPanel />
 						<PostSyncStatus />
-						<PostSticky />
-						<PostPendingStatus />
 						<PostFormat />
 						<PostSlug />
+						<PostTemplatePanel />
 						<PostAuthorPanel />
+						<PostSticky />
+						<PostPendingStatus />
```

Comment:
We have three kinds of panels right now:

 - Some are two column panels with a title and a dropdown on the right
 - Some are select taking the full with like "post author" and the classic theme's "page panel" 
 - Some are toggles like "sticky" and "pending status".
 
 I did a bit of shuffling here to make sure things looks correct (group similar panels together). I think as a follow-up we might want to transform panels like "post author" and "classic page panel" to two columns panels but this PR is very big already, so I just kept these as they are.

Reply:
But they are currently not grouped correctly in trunk?

Reply:
No, it's a bit random.

Reply:
Just worried we are changing the control order for the main editor too, in a refactoring PR. But I don't mind.